### ISSUE TITLE
Adding Aqara hub G3

### ISSUE THREAD

**Issue by SynaZe** [AlexxIT/go2rtc#990]
I am relatively new to Home Assistant and have been learning and exploring its features every day. However, I have encountered a challenge that I am struggling to overcome, and I am reaching out to seek your valuable assistance.

I am attempting to set up my Aqara Hub G3 camera in Home Assistant using WebRTC, and I would greatly appreciate any guidance or support you can provide. Despite my efforts to grasp the intricacies of this process, I find myself facing difficulties.

I setup a integration for the camera through the HomeKit controller, I installed WebRTC camera and I'm now trying to make the camera avialable in my dashboard.

I have this config in the webui:
![image](https://github.com/AlexxIT/go2rtc/assets/48096083/0ec91319-272a-4172-b4c5-54e7d1f52238)

I see the camera under add -> Home Assistant
![image](https://github.com/AlexxIT/go2rtc/assets/48096083/f1e94a0c-8890-4110-8a9a-b38154d09cd2)

But if I want to see the stream of camera, the view remains black
![image](https://github.com/AlexxIT/go2rtc/assets/48096083/e04e635d-a61c-4f40-adfb-938b184407eb)

The error I keep seeing is this one, but I don't know how to solve it or what it even means, I'm quite new to this.
![image](https://github.com/AlexxIT/go2rtc/assets/48096083/afe3a4b1-11f7-4448-b7a1-d6ef00883a8e)

**Comment by AlexxIT** [comment]
No free streams is a "normal error". Most HomeKit cameras support only two simultaneous connections.

**Comment by SynaZe** [comment]
Thank you for the fast response Alexx. With two simultaneuous connections you mean for example I have the camera setup now in aqara and homekit in home assistant. So I can't add another connection. Or do you mean I can't have the stream open at more then 2 devices at once?

**Comment by AlexxIT** [comment]
It's two simultaneous video streams from camera.
One go2rtc stream can make one request.
Hass can't make this request (doesn't support streaming from HomeKit).
If you setup HomeKit proxy via go2rtc - all iOS clients also can make this requests.

**Comment by sdavides** [comment]
I gave up with aqara g3 the video always freezes.
I ran the hack and removed the rtsp authentication, now with the link rtsp://192.168.1.52:8554/360p /720p /1080p /1296p the video is Perfect

**Comment by ricargr** [comment]
> I gave up with aqara g3 the video always freezes. I ran the hack and removed the rtsp authentication, now with the link rtsp://192.168.1.52:8554/360p /720p /1080p /1296p the video is Perfect
> 
> or you can see user and pass from agetprop sys.camera_rtsp_url command from telnet: user=26 pass=83 (change every boot) `~ # agetprop sys.camera_rtsp_url
> 
> {"360p":"rtsp://26:83@192.168.1.4:8554/360p","720p":"rtsp://26:83@192.168.1.4:8554/720p","1080p":"rtsp://26:83@192.168.1.4:8554/1080p","1296p":"rtsp://26:83@192.168.1.4:8554/1296p"}`

Hi sdavides, can you explain how you removed rstp authentication? I have a new g3 and can’t activate telnet on it. Thanks

**Comment by sdavides** [comment]
See at bottom

https://github.com/sdavides/AqaraPOST-Homeassistant

**Comment by ricargr** [comment]
Thank you, sorry for the question i feel like a noob but I cant get the Token and de APPID in the aqara APP. Could you help please

**Comment by sdavides** [comment]
yes, but they are used for other things, not telnet or rtsp authentication. everything is written in the pdf guides

**Comment by ricargr** [comment]
Thanks sdavides, I got it working. One question, I am not able to apply hack on hub g3 that comes with 4.0.2 firmware. I installed 3.3.4 firm but cant scan any qr code. Is there any other way to activate telnet on it? Thanks

**Comment by sdavides** [comment]
no there are no other ways.
Have you generated the QR code?
The image does not have to be too large

**Comment by Lolo3185** [comment]
Hello sdavides,
I made all your integration here : https://github.com/sdavides/AqaraPOST-Homeassistant and everything is fine, so thank you a lot. 
I have just one last question for the video because i do not understand. 
With this integration, could you confirm that we need the firmware 3.3.4?

**Comment by Lolo3185** [comment]
Sorry, I did not see the hack section...
I will try this.
